docker/dev/xdebug.ini

```diff
@@ -0,0 +1,7 @@
+[xdebug]
+zend_extension=xdebug.so
+xdebug.mode=debug
```

Comment:
This is a good start, but is also naive since it only supports debugging and not tracing or development helpers. I suggest this should be set to `develop,debug` for now, and `develop,debug,trace` once sane trace defaults are configured.